x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/Waterfall/waterfall_item.tsx

```diff
@@ -216,27 +211,69 @@ export function WaterfallItem({
         </SpanActionToolTip>
         <HttpStatusCode item={item} />
         <NameLabel item={item} />
-        {errorCount > 0 && item.docType === 'transaction' ? (
-          <ErrorOverviewLink
-            serviceName={item.doc.service.name}
-            query={{
-              kuery: `${TRACE_ID} : "${item.doc.trace.id}" and transaction.id : "${item.doc.transaction.id}"`,
-            }}
-            color="danger"
-            style={{ textDecoration: 'none' }}
-          >
-            <EuiToolTip content={tooltipContent}>
-              <ErrorCount count={errorCount} />
-            </EuiToolTip>
-          </ErrorOverviewLink>
-        ) : null}
+
+        <EventOutcome
+          item={item}
+          transactionErrorCount={transactionErrorCount}
+        />
+
         <Duration item={item} />
         {item.docType === 'span' && <SyncBadge sync={item.doc.span.sync} />}
       </ItemText>
     </Container>
   );
 }
 
+function EventOutcome({
+  item,
+  transactionErrorCount,
+}: {
+  item: IWaterfallSpanOrTransaction;
+  transactionErrorCount: number;
+}) {
+  const theme = useTheme();
+  if (item.doc.event?.outcome !== 'failure') {
+    return null;
+  }
+
+  if (item.docType === 'transaction') {
+    return (
+      <ErrorOverviewLink
+        serviceName={item.doc.service.name}
+        query={{
+          kuery: `${TRACE_ID} : "${item.doc.trace.id}" and ${TRANSACTION_ID} : "${item.doc.transaction.id}"`,
+        }}
+        color="danger"
+        style={{ textDecoration: 'none' }}
+      >
+        <EuiToolTip
+          content={i18n.translate(
+            'xpack.apm.transactionDetails.transactionFailure',
+            {
+              values: { transactionErrorCount },
+              defaultMessage:
+                '{transactionErrorCount, plural, one {View 1 related error} other {View # related errors}}',
+            }
+          )}
+        >
+          <ErrorCount count={transactionErrorCount} />
+        </EuiToolTip>
+      </ErrorOverviewLink>
+    );
+  }
+
+  return (
+    <EuiToolTip
+      content={i18n.translate('xpack.apm.transactionDetails.spanFailure', {
+        defaultMessage:
+          'This span is marked as failed (`event.outcome=failure`)',
```

Comment:
```suggestion
          'This span is marked as failed (event.outcome=failure)',
```
Don't think the tooltips support code blocks like this.